directories/src/redux/epics/node-epics.js

```diff
@@ -143,9 +147,34 @@ const requestGenesisHashEpic = (action$, store) => {
             brokerNode.completeGenesisHashPoW(txid, trytesArray[0])
           ).map(({ data }) => {
             const { purchase: genesisHash, numberOfChunks } = data;
-            return nodeActions.addGenesisHash({ genesisHash, numberOfChunks });
-          })
+            return { genesisHash, numberOfChunks };
+          }).catch(error => {
+            console.log("GENESIS HASH COMPLETE ERROR", error);
+            return Observable.empty();
+          });
         )
+        .mergeMap(({ genesisHash, numberOfChunks }) => {
```

Comment:
lets put this in a separate epic

i think it's because eventually we want to collect a bunch of genesis hashes first before we start treasure hunting, not right after we receive each one.
 
1) create a action for nodeActions.TREASURE_HUNT, `{ type: nodeActions.TREASURE_HUNT payload: { genesisHash, numberOfChunks }`
2) create an epic that listens for it, and paste this stuff in

Reply:
Thats good point, will be done